CHANGES.md

```diff
@@ -0,0 +1,13 @@
+# Changes
+
+## Version 2.0
```

Comment:
Are we already moving towards a new major version? I.e, are there any non-bwc changes in the list. I suppose the filename changes qualify...?

With semantic versioning, shouldn't there be 3 positions in the version number?

Reply:
pom files indicate we're moving towards version 1.0.1

Reply:
Changed pom versions to 2.0.0 & updated the changlog with 2.0.0